esmvaltool/recipes/recipe_combined_indices.yml

```diff
@@ -19,37 +20,35 @@ datasets:
   #- {dataset: IPSL-CM5A-MR, type: exp, project: CMIP5,  mip: day,  exp: historical,  ensemble: r1i1p1,  start_year: 1981,  end_year: 2000}
 
 preprocessors:
-  preproc:
+  preproc1:
     regrid:
       target_grid: MPI-ESM-MR
       scheme: linear
-    mask_fillvalues:
-      threshold_fraction: 0.95
     extract_region:
-      end_latitude: 70
-      start_latitude: 20
-      end_longitude: 300
-      start_longitude: 200
-
+      end_latitude: 90
+      start_latitude: -90
+      end_longitude: 360
+      start_longitude: 0
+   
 diagnostics:
   combine_indices:
     description: Combine indices and/or compute area weighted averages.
     variables:
-      pr:
-        preprocessor: preproc
+      psl:
+        preprocessor: preproc1
```

Comment:
I think the preprocessor can be be removed? Regridding does not make sense for a single model and extracting the region which covers the entire planet does not do much either.